vector/v.db.select/main.c

```diff
@@ -67,12 +95,15 @@ int main(int argc, char **argv)
     char *fsep, *vsep;
     struct bound_box *min_box, *line_box;
     int i, line, area, init_box, cat, field_number;
+    enum OutputFormat format;
+    int vsep_needs_newline;
```

Comment:
You may declare `vsep_needs_newline` as a `bool` if you like, to increase readability.

Reply:
And if you take that path you might as well change `init_box` and `first_rec` too.

Reply:
> You may declare `vsep_needs_newline` as a `bool` if you like, to increase readability.

Done in aec6e015d.

> And if you take that path you might as well change `init_box` and `first_rec` too.

Done in 65ebe227a.

```
 __________
| stdbool! |
 ----------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
_\|/__\|/__\|/_ ||     ||  _\|/__\|/_
```